loops.js

```diff
@@ -51,3 +66,14 @@
 // >> 9
 // ==========================================
 
+for (let i = 0; i < 10; i++) {
+    if (i < 3) {
+        console.log(i);
+    }
+    else if (i > 2 && i  < 7) {
+        console.log("> " + i);
+    }
+    else if (i > 6) {
+        console.log(">> " + i);
+    }
+}
```

Comment:
de code werkt op deze manier maar kan wat verwarrend zijn om te lezen omdat je gebruik maakt van < 3, > 2, < 7 en > 6.

Door hier < 3 , >= 3 en < 7 , >= 7 dan word het wat sneller leesbaar in een oogopslag.